domains/micro.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "MicroBotCodes",
+        "email": "vinceayenni@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "https://website-one-bice.vercel.app/"
```

Comment:
Invalid CNAME. CNAME cannot contain `https://` or `/`. Also, we don't support Vercel unfortunately :(